src/EventSubscriber/OgEventSubscriber.php

```diff
@@ -91,6 +91,12 @@ public function provideDefaultOgPermissions(PermissionEventInterface $event) {
         'description' => $this->t('Edit the group entity.'),
         'default roles' => [OgRoleInterface::ADMINISTRATOR],
       ]),
+      new GroupPermission([
+        'name' => OgAccess::DELETE_GROUP_PERMISSION,
```

Comment:
Do we care about the order? "D"elete comes before "U"pdate in most other instances.

Reply:
OK I will put them alphabetically. I was torn between alphabetical ordering and "CRUD orderding".